docs/Makefile

```diff
@@ -0,0 +1,18 @@
+PROMETHEUS_PUSH_GATEWAY?="http://localhost:9091"
+
+all:
+
+update: update-metrics
+
+# To update Metrics docs:
```

Comment:
Huh, can we have a script/target to do all these steps?

Reply:
`make update-docs`